exercise_010_split_brain_resolver_keep_oldest/src/test/resources/logback-test.xml

```diff
@@ -18,17 +18,17 @@
     
     <logger name="akka" level="ERROR" />
 
-    <logger name="org.neopixel" level="info" additivity="false">
+    <logger name="com.lightbend.training.coffeehouse" level="warn" additivity="false">
```

Comment:
is this intentional?

Reply:
@kikiya 
> is this intentional?

No, that's a mistake. Probably got carried over from the recent merge of a PR that had this content.
Actually, logging to a file makes little sense in the context of this project: workshop participants or, in general, people running this project, will rarely look at the log files.
I updated the PR to not log to a file (it also needlessly wears out the SD cards on a Pi). I did keep a "backup" sample logback config file that logs to a file in exercise 1 (`exercise_001_cluster_base/src/main/resources/logback.xml.toFile`). This can come in handy in case one needs to collect log files in case of a problem.
WDYT?